docs/design/coreclr/jit/hot-cold-splitting.md

```diff
@@ -0,0 +1,173 @@
+# Hot/Cold Code Splitting
+
+This document describes the current state of hot/cold splitting in the JIT.
+
+Hot/Cold splitting is a PGO optimization where generated code is split into frequently-executed ("hot") and
+rarely-executed ("cold") parts and placed in separate memory regions. Increased hot code density better leverages
+spatial locality, which can improve application performance via fewer instruction cache misses, less OS paging, and
+fewer TLB misses.
+
+Hot/Cold splitting in the JIT was previously implemented for AOT-compiled NGEN images in .NET Framework. With hot/cold
+splitting support in Crossgen2 [in progress](https://github.com/dotnet/runtimelab/tree/feature/hot-cold-splitting)
+(and no existing support for splitting dynamically-generated code), the feature has gone untested in the JIT since
+retiring .NET Framework -- as such, regressions have surely been introduced. Furthermore, this existing implementation
+has several major limitations; functions with certain features, like exception handling or switch tables, cannot be
+split yet. Finally, with ARM64/x64 performance parity being a high priority for CoreCLR, it is key hot/cold splitting
+is implemented for ARM64 code generation, too.
+
+The below sections describe various improvements made to the JIT's hot/cold splitting support.
+
+## Testing the JIT Without Runtime Support
+
+Without runtime support for hot/cold splitting in .NET Core yet, testing the JIT's existing hot/cold splitting support
```

Comment:
```suggestion
Without runtime support for hot/cold splitting in .NET as of summer 2022, testing the JIT's existing hot/cold splitting support
```
Nit: We have retired `.NET Core`  as the name a few years ago. We are calling the product just `.NET` now.